src/lattice.py

```
@@ -393,14 +393,14 @@ def _normalize_attention_field(self):
             for node in self.nodes:
                 node.attention /= total_attention
     
-    def update(self, delta_time: float, mouse_influence: Optional[Dict] = None):
+    def update(self, delta_time: float):
```

Ruff (0.14.4)

Unused method argument: `delta_time`

(ARG002)